gravitee-am-repository/gravitee-am-repository-mongodb/src/main/java/io/gravitee/am/repository/mongodb/management/MongoAccountAccessTokenRepository.java

```diff
@@ -40,9 +43,11 @@
 import static com.mongodb.client.model.Filters.eq;
 
 @Repository
+@RequiredArgsConstructor
 public class MongoAccountAccessTokenRepository extends AbstractManagementMongoRepository implements AccountAccessTokenRepository {
     public static final String COLLECTION_NAME = "account_access_tokens";
 
+    private final OrganizationUserRepository organizationUserRepository;
```

Comment:
**TODO**: please do no inject repository into another repository. The search user should be done once in service layer not duplicate into the JDBC & Mongo repository layer